### PR TITLE
Provide proper documentation for non-mutating functions

### PR DESCRIPTION
**Pull request by jon-tow**
A quick clean up to help with #160 to avoid wasting time.

### CODE REVIEW

**Comment by Shashi456** [comment]
I think [this](https://github.com/tensorflow/swift-apis/pull/260#discussion_r295137460) still needs to be addressed.

**Comment by jon-tow** [comment]
Ah sorry, I thought I'd save you some time with this. Should I close this?

**Comment by Shashi456** [comment]
@jon-tow no, this is good work. We can discuss some things about the variety of functions in Math.swift, this does save time.

File: Sources/TensorFlow/Operators/Math.swift [hunk 1]
```diff
@@ -619,7 +619,7 @@ internal func _vjpTan<T: TensorFlowFloatingPoint>(
     return (value, { v in v * (1 + value.squared()) })
 }
 
-/// Computes `sinh` of the specified tensor element-wise.
+/// Returns `sinh` of the specified tensor element-wise.
```
Comment (Shashi456) [inline Sources/TensorFlow/Operators/Math.swift]: @rxwei what do you think about 
```suggestion
/// Returns the hyperbolic sine of the specified tensor element-wise.
```
and similarly for all hyperbolic functions

Comment (jon-tow) [reply Sources/TensorFlow/Operators/Math.swift]: I see what your saying! Sounds good. I'll go through those and give a bit more explanation where possible.

**Comment by jon-tow** [comment]
I confirm the co-authoring!

**Pull request merged**